library/CHART-TEMPLATE/tests/common_cluster_test.yaml

```diff
@@ -1,6 +1,7 @@
 suite: test cluster helper
 templates:
-  - templates/deployment.yaml
+  - templates/integration-deployment.yaml
+  - templates/sidecar-deployment.yaml
```

Comment:
why do we need to add a sidecar template and to test it? to show how to use it?

Reply:
Because tests can change because of things like `.spec.template.spec.containers[?].*`

Also the thought that we had about having a style guide on how to write Helm charts applies in the Chart template.